packages/redux-dynostore-core/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Enhance Redux stores to allow additional functionality to be dynamically added at runtime",
   "author": "Michael Peyper",
   "contributors": [
-    "Jonathan Peyper"
+    "Jonathan Peyper",
+    "Jake Turner"
```

Comment:
Do you mind if I remove this change? I'd like to keep this list as the "maintainers" of the packages and use `all-contributors` to keep track of the contributors.

Reply:
Yep, no worries! 👍